Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+COPY _output /usr/local/bin
+# no tool is more important than others
+ENTRYPOINT ["/bin/sh"]
```

Comment:
can we use CMD here, because if you will want to run some command under the container you will need to override the entrypoint.

Reply:
yep, better to use CMD. Thanks!